uber/templates/preregistration/paid_preregistrations.html

```diff
@@ -50,22 +49,23 @@
 <p>The following preregistrations have been made from this computer:</p>
 
 <table class="table footable">
-  <tr><td>Legal Name</td><td>Badge Name</td><td>Badge Number</td><td>Confirmation Link</td></tr>
```

Comment:
What this table displays probably needs to be configurable via plugin, since what attendees will care about will change from event to event (AC attendees are _very_ interested in their badge number, for example).